[11.0][ADD] account_invoice_validation_queued: Enqueue account invoice validation (#589)
OCA/account-invoicing: account_invoice_validation_queued/readme/DESCRIPTION.rst

This module allows to enqueue in several jobs the account validation process

> to be executed in paralell on background, that's it's normally done

Reviewer: ```suggestion
to be executed in paralell on background, which is normally done
```
